test/moduleTestCases/urls/expected.css

```diff
@@ -2,7 +2,7 @@
 	background: url({./module});
 	background: url({./module});
 	background: url("{./module module}");
-	background: url('{./module module}');
+	background: url("{./module module}");
```

Comment:
`I assume other loaders will make similar choices` - If there is one thing I have learned about this ecosystem, that assumption is almost guaranteed to break something.

Reply:
Right, but "break something" here means "result in a slightly longer string since double-quotes are escaped". The whole point of this change is to make it impossible for other loaders to break your CSS.

My arguments for using only double-quotes summed up:

1. Double-quote wrapping is the expected default for URLs in HTML in all the view-engines I'm familiar with, so I think it is a fair assumption that most loaders will expect this and avoid double-quotes. If they don't, nothing breaks.
2. A user shouldn't have to consider this detail. My own sass-lint rules force single-quotes since this is used across the project, which would work against `svg-url-loader`'s change from double to single-quotes.
3. I don't think the quotes written by the user ought to be considered equal to the quotes produced in the output. The user quotes the module-name, the loader quotes the actual URL, there's no reason to expect similar requirements.

Of course it is not my package, and I'm not that experienced with writing loaders, so I defer entirely to your judgements. If I have failed to convince you, I'll change it so there is a separate escape-function for single-quotes. If that is the case, should I also make an unquoted escape-function?

Reply:
I'm no debating `correctness` I'm just saying that I can all but guarantee that at some point after this lands in master, it will have broken something, somewhere. 

That has nothing to do with not merging it, just saying it's going to happen & we now we have context as to why.

Reply:
Try to keep in mind that `css-loader` sits at the core of the entire style chain and in some places, that's a long & convoluted chain for better or worse.

Anything that lands in this repo has to be vetted, pondered, picked apart & poked so we at the very least can have a reason why there are 4 new issues with version `0.0.x` broke `Y`

Changes in behaviors get that & the rubber glove treatment, it's just too easy to not account for a path in this vast ecosystem where Change X has an unintended side effect on Y  & people show up with torches & pitch forks.